### PR TITLE
fix some content errors

### PR DESCRIPTION
**Pull request by yin1999**
### Description

fix some simple content errors, cherry picked from #12987.

### CODE REVIEW

File: files/zh-tw/web/css/_colon_lang/index.md [hunk 1]
```diff
@@ -5,22 +5,19 @@ slug: Web/CSS/:lang
...
+**`:lang()`** 是一種 [CSS](/zh-TW/docs/Web/CSS) [偽類（pseudo-class）](/zh-TW/docs/Web/CSS/Pseudo-classes)。它會根據括號中指定的語系，來選擇使用的元素。
 
 > **備註：** 在 HTML 中，語言是通過 [`lang`](/zh-TW/docs/Web/HTML/Global_attributes#lang) 屬性和 {{HTMLElement("meta")}} 元素的組合來決定的, 也可能是通過協議的信息來確定 (例如 HTTP 表頭). 對於其他文檔類型，也可能存在其他用於確定語言的方法。
 
-## 語法 Syntax
+## 語法
 
 ### 語法格式
 
-{{csssyntax}}
```
Comment (yin1999) [inline files/zh-tw/web/css/_colon_lang/index.md:23]: see https://developer.mozilla.org/en-US/docs/Web/CSS/:lang

File: files/zh-tw/web/css/animation-fill-mode/index.md [hunk 2]
```diff
@@ -28,9 +28,7 @@ animation-fill-mode: both, forwards, none;
 - `none`
   - : 不执行动画时，动画不会应用任何样式。该元素会使用其 **`keyframes`** 关键动画帧规则来显示动画。这是默认值。
 - `forwards`
-  - : 动画执行后停留到最后一个关键帧动画的计算值，简单来说执行结束动画会停留到结束时候的状态，不会是动画执行之前的效果。比如一个元素从左往右运动，添加当前属性值 **`forwards`** 元素会停留到动画执行结束后的右边而不是最初的左边。可以看以下图
-
-    ![](https://gitee.com/yoonasy/externalLink/raw/master/animation-fill-mode_forwards.gif)
```
Comment (yin1999) [inline files/zh-tw/web/css/animation-fill-mode/index.md:33]: those images are broken, and not exist in en-US docs, removing them.

File: files/zh-tw/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md [hunk 3]
```diff
@@ -664,5 +664,3 @@ We can control the order in which items stack up by using the `z-index` property
...
...<li><a href="/zh-TW/docs/Web/CSS/grid-row-end">grid-row-end</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row-gap">grid-row-gap</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-row-start">grid-row-start</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template">grid-template</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-areas">grid-template-areas</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-columns">grid-template-columns</a></li><li><a href="/zh-TW/docs/Web/CSS/grid-template-rows">grid-template-rows</a></li></ol></li><li data-default-state="open"><a href="#"><strong>Glossary</strong></a><ol><li><a href="/zh-TW/docs/Glossary/Grid">Grid</a></li><li><a href="/zh-TW/docs/Glossary/Grid_lines">Grid lines</a></li><li><a href="/zh-TW/docs/Glossary/Grid_tracks">Grid tracks</a></li><li><a href="/zh-TW/docs/Glossary/Grid_cell">Grid cell</a></li><li><a href="/zh-TW/docs/Glossary/Grid_areas">Grid areas</a></li><li><a href="/zh-TW/docs/Glossary/Gutters">Gutters</a></li><li><a href="/zh-TW/docs/Glossary/Grid_Axis">Grid Axis</a></li><li><a href="/zh-TW/docs/Glossary/Grid_rows">Grid row</a></li><li><a href="/zh-TW/docs/Glossary/Grid_column">Grid column</a></li></ol></li></ol></section>
```
Comment (yin1999) [inline files/zh-tw/web/css/css_grid_layout/basic_concepts_of_grid_layout/index.md:668]: This is a sidebar content (we used long long ago), remove this as we have one at the top of this document.

**Review by jasonren0403 — APPROVED** [review]
Thanks

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**